pep-0685.rst

```diff
@@ -162,17 +163,12 @@ Rejected Ideas
 Using setuptools 60's normalization
 -----------------------------------
 
-Initially this PEP proposed following setuptools to try and minimize
-backwards-compatibility issues.
-But after checking various wheels on PyPI,
+Initially, this PEP proposed using setuptools ``safe_extra()`` for normalization
+to try to minimize backwards-compatibility issues.
+However, after checking various wheels on PyPI,
 it became clear that standardizing **all** naming on :pep:`508` and
-:pep:`503` semantics was easier and better long-term.
-
-
-Open Issues
-===========
-
-N/A
```

Comment:
Why specifically take this out? I left it there as I can never remember where to insert the "Open Issues" section when I end up having to create one.

It's not worth holding this PR up for, though.

Reply:
Ah, I see. I removed it because it was empty and it didn't occur to me that you might forget where to put it, but there's no real harm keeping it while the PEP is still under discussion. If it seems there are likely to be some at this stage, I'm happy to add it back before merging.

Reply:
I added it back, thanks; no harm having it here while the PEP is still in draft.